### ISSUE TITLE
tflint - No such file or directory 

### ISSUE THREAD

**Issue by vitali-shcharbin** [antonbabenko/pre-commit-terraform#80]
Hello, for my typical terraform + terragrunt repo with following .tflint.hcl configuration:
```
config {
  module     = true
  deep_check = false
  force      = false
}
```

And following hook configuration:
```
  - repo: git://github.com/antonbabenko/pre-commit-terraform
    rev: v1.21.0
    hooks:
      - id: terraform_fmt
      - id: terraform_validate
      - id: terraform_tflint
        args: [' --config .tflint.hcl']
      - id: terragrunt_fmt
```

Getting following errors:
```
/Users/user/.cache/pre-commit/repo31phc1no/terraform_tflint.sh: line 42: pushd --config .tflint.hcl modules/dynamodb-table/dynamodb-table.tf modules/dynamodb-table/dynamodb-table-outputs.tf modules/s3-state-bucket: No such file or directory
```

**Comment by grzegorzlisowski** [comment]
(Copy of the comment I put in other issue). To make it work you need two things:

1. Following PR merged into master and new version released (or have your own fork): https://github.com/antonbabenko/pre-commit-terraform/pull/77

2. One of the latest [PRs](https://github.com/antonbabenko/pre-commit-terraform/pull/75) merged to the mainline changed the way of passing arguments to the tflint. 

I have no clue how this is working for anyone: https://github.com/antonbabenko/pre-commit-terraform#notes-about-terraform_tflint-hooks

```hooks:
  - id: terraform_tflint
    args: ['--deep']```

Maybe I'm missing something important. Per the code in the the script and according to the way how pre-commit passes arguments to the scripts (https://pre-commit.com/#passing-arguments-to-hooks), the only solution which I see is this:

```hooks:
  - id: terraform_tflint
    args: ['--args', '--module']```

**Comment by vitali-shcharbin** [comment]
When passing config:
```
args: ['--args', '--config=.tflint.hcl']
```

Getting another issue:
```
dirname: illegal option -- -
usage: dirname path
```

**Comment by grzegorzlisowski** [comment]
Look at point #1 in my comment. This PR must be merged first to master: https://github.com/antonbabenko/pre-commit-terraform/pull/77
OR you could fork this repo and apply the patch on your own.

**Comment by antonbabenko** [comment]
[v1.25.0](https://github.com/antonbabenko/pre-commit-terraform/releases/v1.25.0) of `pre-commit-terraform` has been just released after #77 was merged.
